Gruntfile.js

```diff
@@ -0,0 +1,84 @@
+module.exports = function (grunt) {
+
+  const sass = require('node-sass');
+  const glob = require('glob');
+
+  function getIncludeFiles() {
+    var patterns = [
+      'docroot/themes/humsci/**/*.scss',
+      'docroot/modules/humsci/**/*.scss',
+      'node_modules/decanter/scss',
+      'node_modules/bourbon/core',
+      'node_modules/bourbon-neat/app/assets/stylesheets',
+      'node_modules/neat-omega/core'
+    ];
+
+    var libraries = [];
+    patterns.map(function(pattern){
+      glob.sync(pattern).map(function(file){
+        libraries.push(file);
+      });
+    });
+
+    return libraries;
+  }
+
+  grunt.initConfig({
```

Comment:
We have also become somewhat reliant on the autoprefixer add on. https://github.com/SU-SWS/decanter/blob/master/Gruntfile.js#L56 

It would be nice if you could add that here.